tests/python_tests/test_generation_config.py

```diff
@@ -49,20 +64,18 @@
 @pytest.mark.nightly
 def test_valid_configs(generation_config_kwargs):
     config = GenerationConfig(**generation_config_kwargs)
-    config.validate()
+    verify_set_values(config, generation_config_kwargs)
 
     config = GenerationConfig()
     config.update_generation_config(**generation_config_kwargs)
-    config.validate()
+    verify_set_values(config, generation_config_kwargs)
 
 
 invalid_configs = [
     dict(num_return_sequences=0), # no reason to run with empty output
     dict(num_return_sequences=2), # beam search or multimonial is required
     # stop conditions
     dict(), # no stop conditions at all
-    dict(eos_token_id=1), # 'stop_token_ids' does not contain 'eos_token_id'
-    dict(eos_token_id=1, stop_token_ids={2}), # 'stop_token_ids' is not empty, but does not contain 'eos_token_id'
```

Comment:
actually, such configs are not correct.

They become correct after `update_generation_config`, but if use:
```python
GenerationConfig config
config.eos_token_id = 1
config.stop_token_ids={2}
config.validate() # should throw
```

maybe we need to test assignment via fields separately?

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1633